### ISSUE TITLE
Support for automated tests

### ISSUE THREAD

**Issue by ErayCetinerGelato** [beefe/react-native-actionsheet#82]
Is it possible to add accessibility IDs/testIDs to action sheet buttons?

**Comment by isa3bel** [comment]
yes, in ActionSheetCustom.js in the method _createButtons. In TouchableHighlight there's already `accessible=true`, and `accessibilityLabel={title}` you can add `testID={title}`

**Comment by maxime-helen** [comment]
I don't find it 🤔And the snapshot https://github.com/beefe/react-native-actionsheet/blob/master/tests/__snapshots__/ActionSheetCustom.test.js.snap#L217 shows `undefined` passed to `testID`
